### PR TITLE
NLS.md: new file with instructions to use weblate or PR method

### LINKED ISSUE #17: Translations with weblate
I opened an account at https://hosted.weblate.org  for `labwc-tweaks-gtk`

Below you can see it:

https://hosted.weblate.org/projects/labwc-tweaks-gtk/

If you want to translate this project go to:  https://hosted.weblate.org/engage/labwc-tweaks-gtk/ and register.

To do:

- [x] add this info to README.md when @WeblateOrg aproves my account for a free account, which depends on the licence, no issue there
- [x] add the badge for translations
- [x] add message about using PR method for those who don't want to use weblate
- [x] add instructions for code contributors 
- [x] ~add a note to translate `data/labwc-tweaks-gtk.desktop`~ no longer needed, added in #20

The badge:

status badge

@johanmalm @stefonarch

### PR DESCRIPTION
**Pull request by 01micko**
for translation. Also brief instructions for code contributors. README.md: add weblate status badge, fixes #17

### CODE REVIEW

**Comment by 01micko** [comment]
Marked WIP pending weblate approval for free host account.

**Comment by 01micko** [comment]
Also, haven't added the `labwc-tweaks-gtk.desktop` instructions yet because I'm unsure how to fix it in weblate. Working on it.

File: NLS.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+```C
+gtk_label_new(_("My New Feature"));
+
+```
+
+2. If you create a new file that has exposed strings append the _filename_
+to `po/POTFILES.in`
+
+3. The rest of the workflow is that same as in `labwc`.
```
Comment (01micko) [inline NLS.md:30]: typo reminder: s/that/the/

**Comment by 01micko** [comment]
This will now depend on #20 with a fixup commit soon

**Comment by 01micko** [comment]
Apparently this is a prerequisite for approval by weblate. So marking ready for review.

**Comment by johanmalm** [comment]
LGTM

**Pull request merged**